packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -232,10 +233,13 @@ export abstract class RetrieveExecutor<T> extends DeployRetrieveExecutor<T> {
     this.setupCancellation(operation, token);
 
     const result: RetrieveResult = await operation.pollStatus();
-    await SourceTrackingService.updateSourceTrackingAfterRetrieve(
-      sourceTracking,
-      result
-    );
+    const orgType = await getWorkspaceOrgType();
+    if (orgType === OrgType.SourceTracked) {
+      await SourceTrackingService.updateSourceTrackingAfterRetrieve(
+        sourceTracking,
+        result
+      );
+    }
```

Comment:
Does this cover the conflict detection retrieve?

Reply:
Hmm.  VSCE doesn't have conflict detection on retrieve.  Our setting doesn't apply and STL is configured with ignoreConflicts: true.  But, I should probably double check here that the retrieve was successful.

Reply:
cool..yeah I was having to knock some rust off about how that worked exactly.  Another use case to verify here is delete as I saw the retrieve error in that flow on release testing 👍

Reply:
I updated it to only make the call on a successful retrieve operation.  I did some spot testing with deleting files and trying to retrieve them and such from a non-src-tracked org and didn't see any errors.